examples/ul2/src/super_glue/data.py

```diff
@@ -0,0 +1,262 @@
+                'labels': decoder_dict.input_ids,
+                'decoder_attention_mask': decoder_dict.attention_mask,
+            }
+
+    assert isinstance(dataset, datasets.Dataset)
+    columns_to_remove = list(dataset[0].keys())
+    dataset = dataset.map(
+        tokenize_function,
+        batched=False,
+        num_proc=None if num_workers == 0 else num_workers,
+        # batch_size=3, #1000,
+        remove_columns=columns_to_remove,
+        new_fingerprint=
+        f'{task}-{tokenizer_name}-tokenization-{max_seq_length}-{split}',
+        load_from_cache_file=False,
+    )
+    return dataset
+
+
+def _build_dataloader(dataset, cfg, device_batch_size):
+    dataset = cast(Dataset, dataset)
+
+    return DataLoader(
+        dataset,
+        collate_fn=transformers.default_data_collator,
+        batch_size=device_batch_size,
+        sampler=dist.get_sampler(dataset,
+                                 drop_last=cfg.drop_last,
+                                 shuffle=cfg.shuffle),
+        num_workers=cfg.num_workers,
+        pin_memory=cfg.pin_memory,
+        prefetch_factor=cfg.prefetch_factor,
+        persistent_workers=cfg.persistent_workers,
+        timeout=cfg.timeout,
+    )
+
+
+def build_super_glue_task_dataloader(cfg: Mapping[str, Any],
+                                     device_batch_size: int,
+                                     mode: Optional[str] = None):
+
+    assert cfg.name == 'super_glue', f'Tried to build super_glue dataloader with cfg.name={cfg.name}'
+
+    # One task
+    if isinstance(cfg.dataset.task, str):
+        dataset = create_super_glue_dataset(cfg.dataset.task,
+                                            cfg.dataset.tokenizer_name,
+                                            cfg.dataset.split,
+                                            cfg.dataset.max_seq_length,
+                                            cfg.dataset.decoder_only_format,
+                                            extra_prefix=cfg.dataset.get(
+                                                'extra_prefix', None))
+        return _build_dataloader(dataset, cfg, device_batch_size)
+
+    # Task mixture
+    elif isinstance(cfg.dataset.task, (list, tuple, ListConfig)):
+        tasks = [task.lower() for task in cfg.dataset.task]
+
+        if mode not in ['train', 'eval']:
+            raise ValueError(
+                'When using multiple tasks, argument `mode` must be set to either `train` or `eval`.'
+            )
+
+        super_glue_datasets = [
+            create_super_glue_dataset(task,
+                                      cfg.dataset.tokenizer_name,
+                                      cfg.dataset.split,
+                                      cfg.dataset.max_seq_length,
+                                      cfg.dataset.decoder_only_format,
+                                      extra_prefix=cfg.dataset.get(
+                                          'extra_prefix', None))
```

Comment:
```suggestion
            create_super_glue_dataset(
                task,
                cfg.dataset.tokenizer_name,
                cfg.dataset.split,
                cfg.dataset.max_seq_length,
                cfg.dataset.decoder_only_format,
                extra_prefix=cfg.dataset.get('extra_prefix'),
```
(The diff is confused; all I did is change the linebreaks/indentation and remove the `, None` in the `get()`)